### PR TITLE
Restart failed cluster

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

File: pkg/workflows/steps/certificates/certificates.go [hunk 1]
```diff
@@ -45,9 +46,12 @@ func (s *Step) Run(ctx context.Context, out io.Writer, config *steps.Config) err
...
 
 	if !config.IsMaster {
-		master := config.GetMaster()
-		config.CertificatesConfig.MasterHost = master.PrivateIp
-		config.CertificatesConfig.NodeName = config.Node.Name
+		if master := config.GetMaster(); master != nil {
+			config.CertificatesConfig.MasterHost = master.PrivateIp
+			config.CertificatesConfig.NodeName = config.Node.Name
+		} else {
+			logrus.Errorf("master not found in config")
```
Comment (saheienko) [inline pkg/workflows/steps/certificates/certificates.go]: Should we continue in this case?

File: pkg/workflows/steps/config.go [hunk 2]
```diff
@@ -469,15 +474,22 @@ func NewConfigFromKube(profile *profile.Profile, k *model.Kube) (*Config, error)
 		configChan:       make(chan *Config),
 	}
 
-	if k != nil {
-		cfg.Kube = *k
```
Comment (saheienko) [inline pkg/workflows/steps/config.go]: Please, keep this check there are cases when it is `nil`.

File: pkg/workflows/steps/config.go [hunk 3]
```diff
@@ -380,6 +380,9 @@ func NewConfigFromKube(profile *profile.Profile, k *model.Kube) (*Config, error)
 		DigitalOceanConfig: DOConfig{
 			Region: profile.Region,
 		},
+		Kube: *k,
```
Comment (saheienko) [inline pkg/workflows/steps/config.go]: Could this panic with the `nil pointer dereference`?

File: pkg/workflows/steps/certificates/certificates.go (hunk 1 above)
Comment (stgleb) [reply pkg/workflows/steps/certificates/certificates.go]: I agree, lets fail here

File: pkg/workflows/steps/config.go (hunk 3 above)
Comment (stgleb) [reply pkg/workflows/steps/config.go]: I've added a check above

**Pull request merged**